R/utils_render_html.R

```diff
@@ -237,45 +236,29 @@ coalesce_marks <- function(
     locname,
     delimiter = ","
 ) {
-  filtered_tbl <- dplyr::filter(fn_tbl, locname == !!locname)
-  dplyr::summarize(filtered_tbl, fs_id_c = paste(fs_id, collapse = delimiter))
+  fs_ids <- vctrs::vec_slice(fn_tbl$fs_id, fn_tbl$locname == locname)
+  paste(fs_ids, collapse = delimiter)
```

Comment:
Note that I refactored `coalesce_marks()` to return a single string because the resulting data frame (1 col x 1 row) previously was never used, and all code needed to access the fs_id_c variable.

Reply:
This is a great improvement! Yeah, a vector is all we need.